Marlin/temperature.cpp

```diff
@@ -199,7 +199,7 @@ static void updateTemperaturesFromRawValues();
 //================================ Functions ================================
 //===========================================================================
 
-void PID_autotune(float temp, int extruder, int ncycles) {
+void PID_autotune(float temp, int extruder, int ncycles, bool set_result) {
```

Comment:
I've checked and I don't seem to see any other places calling `PID_autotune()` other than in `gcode_M303()`, but as we do not know what other hacks people may have been doing with it, it's wiser not to break behavior by defaulting `set_result` to `false`, this way is still possible to use `PID_autotune(temp, e, c);`.

``` cpp
void PID_autotune(float temp, int extruder, int ncycles, bool set_result /* = false */) {
````